homework/02LessonTwo.js

```diff
@@ -117,7 +155,20 @@ function whichSpecies(character) {
 // return the string "divisible by 2" when number % 2 === 0
 // return the string "divisible by 3" when number % 3 === 0
 // return the string "divisible by 5" when number % 5 === 0
-
+function testNumber (number) {
+  if (number % 4 === 0) {
+    return divisible by 4;
+  }
+  if (number % 2 === 0) {
+    return divisible by 2;
+  }
+  if (number % 3 === 0){
+    return divisible by 3;
+  }
+  if (number % 5 === 0){
+    return divisible by 5;
+  }
```

Comment:
What if the number is NOT divisible by 4,2,3 or 5? What should the function return?